### PR TITLE
Handy updates!

### LINKED ISSUE #272: Unpackrr intermittently fails to delete $movietitle_unpackerred directories
Using Unpackrr on UNRAID 6.11.5 with Hotio's Docker App. And seeing behavior where Unpackrr intermittently fails to delete $movietitle_unpackerred directories. Have not been able to isolate in the logs yet when this is occurring and what the logs are writing when it occurs. 

When it works correctly, I am seeing messages like below. Which speak to it being intermittent.

2023/03/09 12:11:40 Deleted (recursively): /data/torrents/movies/Alpha.2018.1080p.BluRay.x264-DRONES_unpackerred
2023/03/09 12:11:40 Deleted (recursively): /data/torrents/movies/Alpha.2018.1080p.BluRay.x264-DRONES_unpackerred/Subs

When it does not work I am left with several empty directories in the root directory that unpackerr never goes back to clean up.
image

### LINKED ISSUE #325: Unable to unrar multirar files
Im using it to automate the download stack from jellyfin, it supposed to unrar multirar files or is not planned?

### LINKED ISSUE #331: Add folder flag to disable recursion.
hi i have installed unpacker on truenas scale to watch  a folder it works but i have a request if it is possible to add a parameter to only extract downloaded archive and not the archive inside. I use this expecially for retro games and inside the downloaded archive there are a lot o other archive that have to remain compressed. a global parameter called UN_EX_FIRST_LEVEL default false to activate recursion and true to stop it.

Origin user: bmiki75
Origin message: [Discord Server Link](https://discord.com/channels/184509568867368963/781625367391371274/1142221539819933847)
Reporting user: twitchcaptain
Reporting message: [Discord Server Link](https://discord.com/channels/184509568867368963/781625367391371274/1142487932318519300)

### LINKED ISSUE #334: [Request] Delete empty folder
Hello,

Using unpackerr, there's a feature to delete the extracted file after extraction. It would be nice if it could also delete the folder if it's empty. My current configuration extract the file in a temp folder named <torrent name>_unpackerred and then move the file back in <torrentname>. After that, the corresponding *ar program import the extracted file and change the category in qbittorrent. By changing the category, the content of the torrent is then moved to another folder. This leave the extracted file in the folder. Unpackerr then pass 5 minutes after and delete the file but leave the empty folder.

Thank you

### PR DESCRIPTION
**Pull request by davidnewhall**
- Updates ISO package for #264 
- Closes #331
  - Adds a new input for folders that allows disabling archive-in-archive recursion.
- Closes #325
  - This refactors some code to more cleanly and correctly use the `OutputPath` parameter from the Starr Queue API.
- Closes #272
  - Found a bug that leaves behind an empty temporary folder.
- Re-does #304 by moving stderr to a config option. 
  - Hiding errors in std err is problematic for Windows and Portainer (at minimum).
- Closes #334.
  - Attempts to purge an empty folder when cleaning up. Hoping @nodiaque can test.

### CODE REVIEW

**Comment by davidnewhall** [comment]
Download here: https://unstable.golift.io

File: pkg/unpackerr/folder.go [hunk 1]
```diff
@@ -20,15 +20,18 @@ import (
...
-	ExtractISOs bool          `json:"extract_isos" toml:"extract_isos" xml:"extract_isos" yaml:"extract_isos"`
-	Path        string        `json:"path" toml:"path" xml:"path" yaml:"path"`
+	DeleteOrig       bool          `json:"delete_original" toml:"delete_original" xml:"delete_original" yaml:"delete_original"`
+	DeleteFiles      bool          `json:"delete_files" toml:"delete_files" xml:"delete_files" yaml:"delete_files"`
+	DisableLog       bool          `json:"disable_log" toml:"disable_log" xml:"disable_log" yaml:"disable_log"`
+	MoveBack         bool          `json:"move_back" toml:"move_back" xml:"move_back" yaml:"move_back"`
+	DeleteAfter      cnfg.Duration `json:"delete_after" toml:"delete_after" xml:"delete_after" yaml:"delete_after"`
+	ExtractPath      string        `json:"extract_path" toml:"extract_path" xml:"extract_path" yaml:"extract_path"`
+	ExtractISOs      bool          `json:"extract_isos" toml:"extract_isos" xml:"extract_isos" yaml:"extract_isos"`
+	DisableRecursion bool          `json:"disableRecursion" toml:"disable_recursion" xml:"disable_recusion" yaml:"disableRecursion"`
```
Comment (davidnewhall) [inline pkg/unpackerr/folder.go:33]: typo

**Comment by pando85** [comment]
When is this plan to be released? Just to know when to change back to a pinned version. Thanks :)

**Comment by davidnewhall** [comment]
I'll try to get a release out in September.

**Pull request merged**